src/main/app/apply_obligations.xml

```diff
@@ -63,6 +64,15 @@ otherwise
     </sub-flow>
     <sub-flow name="apply_notify">
         <logger message="Apply Notify!!" level="INFO" doc:name="Obligation Notify"/>
+        <choice doc:name="Choice">
+            <when expression="#[flowVars.applyObligations.Result.Decision == 'Deny']">
+                <set-payload value="You don't have  access to #[sessionVars.EntryParams.get(&quot;To&quot;)]" encoding="UTF-8" mimeType="text/plain" doc:name="Set Payload"/>
+                <smtp:outbound-endpoint host="smtp.gmail.com" port="587" user="tesisglm%40gmail.com" password="tesis2016" connector-ref="Gmail" to="juanlacu@gmail.com" from="alejandrodamasco@gmail.com" subject="Deny Service" responseTimeout="10000" doc:name="SMTP"/>
```

Comment:
We should adjust this part, the emails should be in the constants file, and we must replace the hard coded email to one get it from the obligation.